test_aodndata/gsla/test_gsla.py

```diff
@@ -44,14 +34,14 @@ def test_push_unzipped_nc(self):
         """ check on pushing *.nc rather than *.nc.gz """
 
         with TemporaryDirectory() as tmpdir:
-            with gzip.open(GOOD_NC_GZ_DM01, 'rb') as f_in:
-                good_nc_dm01 = os.path.join(tmpdir, os.path.basename(GOOD_NC_GZ_DM01.replace('.gz', '')))
+            with gzip.open(GOOD_NC_GZ_DM02_nc, 'rb') as f_in:
```

Comment:
why not doing this test entirely with the GOOD_NC_GZ_DM02 file instead of GOOD_NC_GZ_DM02_nc. You'll keep one file then. so you can get rid of the variable GOOD_NC_GZ_DM02.nc at the top

Reply:
Yes, good pick - updated in [commit b90ada8](https://github.com/aodn/python-aodndata/pull/342/commits/b90ada896f5f6a2a5a80fb2ae84209edae66e05f)